sqlmesh/core/context.py

```diff
@@ -395,13 +396,16 @@ def run(
             end: The end of the interval to render.
             latest: The latest time used for non incremental datasets.
             skip_janitor: Whether to skip the janitor task.
+            ignore_cron: Whether to ignore the model's cron schedule and run all available missing intervals.
         """
         environment = environment or c.PROD
         self.notification_target_manager.notify(
             NotificationEvent.RUN_START, environment=environment
         )
         try:
-            self.scheduler(environment=environment).run(environment, start, end, latest)
+            self.scheduler(environment=environment).run(
+                environment, start, end, latest, ignore_cron=False
```

Comment:
did you mean to hardcode ignore_cron here?

Reply:
just fixed this, thanks